### PR TITLE
Add Mxnet in ReadMe

### PR DESCRIPTION
**Pull request by beru**
excerpted from #432 which is written by Erwan BERNARD(edmBernard)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,16 +49,21 @@ Check out the [documentation](http://tiny-dnn.readthedocs.io/) for more info.
...
+|Prerequisites|__Nothing__(Optional:TBB,OpenMP)|BLAS,Boost,protobuf,glog,gflags,hdf5, (Optional:CUDA,OpenCV,lmdb,leveldb etc)|Numpy,Scipy,BLAS,(optional:nose,Sphinx,CUDA etc)|numpy,six,protobuf,(optional:CUDA,Bazel)|BLAS(optional:CUDA)|
+|Modeling By|C++ code|Config File|Python Code|Python Code|C++, Python, R, Julia ...|
+|Training|C++|C++, Python|Python|Python|C++, Python, R, Julia ...|
+|Execution|C++|C++, Python|Python|Python, C++*2|C++*4, Python, R, Julia ...|
+|GPU Support|No|Yes|Yes|Yes|Yes|
+|Installing|Unnecessary|Necessary|Necessary|Necessary|Necessary*3|
+|Windows Support|Yes|No*|Yes|Yes|Yes|
+|Pre-Trained Model|Yes(via caffe-converter)|Yes|No*|No*|Yes(via caffe-converter)|
+
+*unofficial version is available
```
Comment (edgarriba) [inline README.md]: use this instead:
 `whateverword<sup>1</sup>` -> whateverword<sup>1</sup>

**Comment by beru** [comment]
Please use [this page](https://github.com/beru/tiny-cnn/tree/add_mxnet_in_readme) for checking.

File: README.md (hunk 1 above)
Comment (beru) [inline README.md]: Followed the suggestion so please check it again.

**Comment by beru** [comment]
> edgarriba wrote: I'm also wondering if we transpose the grid it will fit better?

Maybe so, should I ready another branch to check that layout?

**Comment by edgarriba** [comment]
let's merge this first

**Comment by nyanp** [comment]
It's a good idea to transpose this table. There are too many columns..

File: README.md (hunk 1 above)
Comment (beru) [inline README.md]: @edgarriba Followed the suggestion. Please check it again please.

Comment (edgarriba) [inline README.md]: looks good

**Comment by edgarriba** [comment]
@beru @nyanp talking about style, I think that compared to ohter projects we have an overloaded README file. Probably we could show only basic info and one of the small scripts

**Comment by beru** [comment]
@edgarriba Users can read through the document while they're building the projects.

**Comment by bhack** [comment]
@beru Are you alluding to https://mobile.twitter.com/ernerfeldt/status/777045892171964417? :smile:

**Pull request merged**